containers/kubernetes/reference-content/version-support-policy.mdx

```diff
@@ -89,4 +102,34 @@ The following documentation lists only the main feature changes for each version
 * Exec Probe Timeout Handling
 
 ### 1.19
-Official [release announcement](https://kubernetes.io/blog/2020/08/26/kubernetes-release-1.19-accentuate-the-paw-sitive/)
\ No newline at end of file
+Official [release announcement](https://kubernetes.io/blog/2020/08/26/kubernetes-release-1.19-accentuate-the-paw-sitive/)
+
+## Supported Container Runtime Interfaces (CRI)
+
+### Kubernetes v1.26
+Starting from Kubernetes version 1.26 launch:
+* Kubernetes Kapsule only supports **cilium** and **calico** as CNI.
+* Kubernetes Kosmos only supports **kilo** as CNI.
+
+| Interface | Deprecation |  Obsolete  |
```

Comment:
- We can remove the Deprecation column, as we make Flannel & Weave become obsolete straight from 1.25.
- Let's put 1.25 in the obsolete column.